### ISSUE TITLE
Some suggestions for updating the lesson's home page

### ISSUE THREAD

**Issue by ghar1821** [datacarpentry/R-ecology-lesson#837]
Hi there,

I have a few suggestions for the preparation step of the lesson (https://datacarpentry.org/R-ecology-lesson/index.html):

1. For installing R and Rstudio, I think the Rstudio website has change since the lesson's home page was created. I can no longer find the "Under All Installers" option anymore to download Rstudio installation file. The following screenshot is what I see:
<img width="1262" alt="image" src="https://user-images.githubusercontent.com/5366317/227065085-9f364445-448f-4edd-9188-ea44902a30c5.png">
Depending on what OS you are running, I think the button on the left (Install Rstudio) automatically changes.

2. For downloading R from CRAN website, for Mac OS, there are 2 different `.pkg` files, 1 for the mac with M1 chip (R-4.2.3-arm64.pkg), the other for mac with intel chip (R-4.2.3.pkg). Theoretically, the M1 chip can run the R version designed for the M1 chip using Rosetta, but for learners with Mac running M1 chip, it is perhaps better to use the R arm64 version. Maybe an idea to clarify this?

3.  For the Linux installation, learners whose computers are managed by their organisation's IT department, will most likely not have the permission to run `sudo`. Maybe an idea to highlight this somewhere so they can make sure they seek IT's help to either do the installation or upgrade their account?

4. In the page, it says: 

```
R is a programming language that is especially powerful for data exploration, visualization, and statistical analysis
```

But in the "Before we start" page (https://datacarpentry.org/R-ecology-lesson/00-before-we-start.html), it says:
```
The term “R” is used to refer to both the programming language and the software that interprets the scripts written using it.
```

It maybe better to consolidate them?

5. Is it an idea to move the preparation step content into the "Before we start" page? It may make everything flow better when one is teaching, i.e., start with explaining what R and Rstudio is, then proceed to how to install them, then "knowing your way around Rstudio", and so on. 

6. The step on installing packages. Is it better to move this to the Introduction page? Specifically after the section explaining what functions are? https://datacarpentry.org/R-ecology-lesson/01-intro-to-r.html#Functions_and_their_arguments. Where it is now, a newcomer will most likely not know what packages are, or in fact, may not even know where the console window in Rstudio is located.

I'm happy to make a pull request to address the points above. Hope they help!

**Comment by tobyhodges** [comment]
Thanks @ghar1821 for opening this issue. The lesson underwent a major update and reorganisation when https://github.com/datacarpentry/R-ecology-lesson/pull/887 was merged. As your suggestion relates to content in a version of the lesson before that update took place, I will close it. Please open a new issue if you have suggestions for how the new content could be improved.
